xmalloc.c:135:16: warning: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Wimplicit-function-declaration]
  p = xmalloc (strlen (str) + 1);
               ^
xmalloc.c:135:16: note: include the header <string.h> or explicitly provide a declaration for 'strlen'
xmalloc.c:136:3: warning: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Wimplicit-function-declaration]
  strcpy (p, str);
  ^
xmalloc.c:136:3: note: include the header <string.h> or explicitly provide a declaration for 'strcpy'
--- src/xmalloc.c.orig	2001-05-05 07:40:53.000000000 -0500
+++ src/xmalloc.c	2018-11-05 07:11:31.000000000 -0600
@@ -25,6 +25,7 @@
 # define VOID char
 #endif
 
+#include <string.h>
 #include <sys/types.h>
 
 #if STDC_HEADERS
